sktime/forecasting/base/_fh.py

```diff
@@ -505,7 +505,7 @@ def to_absolute_index(self, cutoff=None):
 
         Returns
         -------
-        fh : ForecastingHorizon
```

Comment:
This was not what was returned! Prob. error in docs.

Reply:
yes, looks like a typo - should be always a `pd.Index`, as the docstring preamble says. Correction is appreciated.